tests/Doctrine/Tests/Models/Translation/DatedComment.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Doctrine\Tests\Models\Translation;
+
+use Doctrine\ODM\PHPCR\Mapping\Annotations as PHPCRODM;
+
+/**
+ * @PHPCRODM\Document(translator="attribute")
```

Comment:
i thought we fixed that the attribute from the parent is inherited. but without this i get an error that "text" is translated but there is no translator strategy